backend/app/record/record_reader/wikipedia_record_reader/wikipedia_record_reader.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING
+
+from app.models.wikipedia_record import WikipediaRecord
+from app.record.record_reader.record_reader import RecordReader
+from app.utils.encodings.encodings_list import encodings
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+
+class WikipediaRecordReader(RecordReader):
+    """A concrete implementation of RecordReader.
+
+    Read in different formats of data and parse them into Wikipedia records.
+    """
+
+    def load_json_data(
```

Comment:
WikipediaRecordReader should read the `dotenv` (or a singleton of it) in its `__init__`. Then the `read` implementation doesn't need to take any parameters.